front/cypress/e2e/admin/users/seat-billing.cy.ts

```diff
@@ -0,0 +1,708 @@
+import { randomString, randomEmail } from '../../../support/commands';
+import { generateProjectFolder } from '../../../fixtures';
+
+describe('Seat based billing', () => {
+  let createdUserIds: string[] = [];
+  let adminAndmoderatorsCount: number;
+
+  type CreateUserType = {
+    firstName: string;
+    lastName: string;
+    email: string;
+    password: string;
+  };
+
+  const createUsers = (users: CreateUserType[]) => {
```

Comment:
```suggestion
  const createUsers = (createUserData: CreateUserType[]) => {
```
Was a bit confused by the "users"